python/operations.cpp

```diff
@@ -119,6 +119,27 @@ template <typename T> void bind_contains_events(py::module &m) {
           .c_str());
 }
 
+template <class T> void bind_reshape(pybind11::module &mod) {
```

Comment:
I suppose these (and concatenate) should go into `python/shape.cpp` to match the naming in the other modules. Could you move them? Cheers!

Reply:
Yes. I'm also exposing `transpose` as that's missing.